vars.md

```diff
@@ -9,6 +9,8 @@ assured that they are persisted.
 
 | Name | Option | Allowed values | Default |
 | :------ | :------- | :--------------- | :-------- |
+| NUM_OF_MASTER_REPLICAS | Set the controlplane replica count. ||1| 
```

Comment:
Should it be ||1| ? There are double |

Reply:
> Should it be ||1| ? There are double |

Third column is empty, so I assume this is correct?

Reply:
Aha ok true